includes/blocks/class-sensei-global-blocks.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * File containing the class Sensei_Global_Blocks.
+ *
+ * @package sensei
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class Sensei_Global_Blocks
+ */
+class Sensei_Global_Blocks extends Sensei_Blocks_Initializer {
+	/**
+	 * Sensei_Global_Blocks constructor.
+	 */
+	public function __construct() {
+		parent::__construct( null );
+	}
+
+	/**
+	 * Initialize blocks that are used in course pages.
+	 */
+	public function initialize_blocks() {
+		new Sensei_Learner_Courses_Block();
+	}
+
+	/**
+	 * Enqueue frontend and editor assets.
+	 *
+	 * @access private
+	 */
+	public function enqueue_block_assets() {
+		Sensei()->assets->enqueue_style( 'sensei-shared-blocks-style' );
```

Comment:
I thought about create another category of assets for only global blocks, so we could have smaller assets in post types that don't use any other block, like in the default posts, for example. Using the shared assets, we'll also include there the assets we use shared between courses and lessons only, for example.

But to avoid more complexity, I decided to include it in the shared for now.

Reply:
See https://github.com/Automattic/sensei/pull/4122#discussion_r601833048